environment_docs.yaml

```diff
@@ -22,4 +22,4 @@ dependencies:
   - sphinx-book-theme
   - nbsphinx
   - nbsphinx_link
-  - sphinxcontrib-mermaid
+  - git+https://github.com/maurerle/sphinxcontrib-mermaid@maurerle
```

Comment:
why do we need to rely on this custom version?

Reply:
The initialization of mermaid js is currently broken on the upstream version - see here:
https://github.com/assume-framework/assume/issues/272